Taskfile.yaml

```
@@ -147,3 +150,36 @@ tasks:
     cmds:
       - task: cve:bin
       - tools/cve/scan-main.sh {{.REPORT_FILE_NAME}}
+
+  dlv:virt-controller:build:
+    desc: "Build image virt-controller with dlv"
+    cmd: docker build -f ./images/virt-controller/debug/dlv.Dockerfile -t "{{ .DLV_IMAGE }}" .
+
+  dlv:virt-controller:build-push:
+    desc: "Build and Push image virt-controller with dlv"
+    cmds:
+      - task: dlv:virt-controller:build
+      - docker push "{{ .DLV_IMAGE }}"
+      - task: dlv:virt-controller:print
+
+  dlv:virt-controller:print:
+    desc: "Print subcommands for debug"
```

What is the subcommand?

```suggestion
    desc: "Print commands for debug"
```